EyeOfTruth.Shared/LaMulanaClassic.cs

```diff
@@ -0,0 +1,89 @@
+﻿using LiveSplit.ComponentUtil;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+
+namespace EyeOfTruth
+{
+    public class LaMulanaClassic : WatchedProcess
```

Comment:
I thought I got rid of this file, on account of not supporting classic and it being more confusing to have all the code for classic stuff hanging around. Did you add it back in?